Dockerfile

```diff
@@ -1,5 +1,17 @@
+FROM golang:alpine as builder
+
+WORKDIR /go/src/github.com/blendle/pg2kafka
+ADD . ./
+
+RUN apk --update --no-cache add git alpine-sdk bash
+RUN wget -qO- https://github.com/edenhill/librdkafka/archive/v0.11.4-RC1.tar.gz | tar xz
```

Comment:
I would group some of these statements into one layer

Reply:
I don't, it's a builder pre-image, not used in the final image.